File: AdminArea/AdminArea.Web/Modules/Reviewcom/Reviewcom/ReviewcomForm.cs
using Serenity.ComponentModel;

namespace AdminArea.Reviewcom.Forms;

[FormScript("Reviewcom.Reviewcom")]
[BasedOnRow(typeof(ReviewcomRow), CheckNames = true)]
public class ReviewcomForm
{
    public string Name { get; set; }
    public string Email { get; set; }
    public string Subject { get; set; }
    public string Message { get; set; }
}

File: AdminArea/AdminArea.Web/Modules/ServerTypes/Reviewcom/ReviewcomForm.ts
import { StringEditor, PrefixedContext, initFormType } from "@serenity-is/corelib";

export interface ReviewcomForm {
    Name: StringEditor;
    Email: StringEditor;
    Subject: StringEditor;
    Message: StringEditor;
}

export class ReviewcomForm extends PrefixedContext {
    static readonly formKey = 'Reviewcom.Reviewcom';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!ReviewcomForm.init)  {
            ReviewcomForm.init = true;

            var w0 = StringEditor;

            initFormType(ReviewcomForm, [
                'Name', w0,
                'Email', w0,
                'Subject', w0,
                'Message', w0
            ]);
        }
    }
}